experimental/dds/tree2/src/feature-libraries/editable-tree-2/lazyTree.ts

```diff
@@ -296,7 +296,13 @@ export class LazyMap<TSchema extends MapSchema>
 		makePropertyEnumerableOwn(this, "asObject", LazyMap.prototype);
 	}
 
-	public get(key: FieldKey): TypedField<TSchema["mapFields"]> {
+	public get(key: FieldKey): UnboxField<TSchema["mapFields"]> {
```

Comment:
Nit: Adding the {@inheritDoc` comments to these improves intellisense in some cases. And if we ever end up exporting this type from the package, it would ensure the write docs get generated.

Reply:
Good call, I'll start doing that (when I remember :D).